R/makephyloseqFromTreeSummarizedExperiment.R

```diff
@@ -0,0 +1,113 @@
+#' Create a phyloseq object from a TreeSummarizedExperiment object
+#'
+#' @param x a \code{TreeSummarizedExperiment} object
+#'
+#' @param abund_values
+#' A single character value for selecting the
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}}
+#'   to be included in the phyloseq object that is created.
+#'
+#'  @details
+#' \code{makePhyloseqFromTreeSummarizedExperiment} is used for creating a phyloseq
+#' object from TreeSummarizedExperiment object.
+#'
+#' @return
+#' An object of class \code{Phyloseq} object.
+#'
+#' @name makePhyloseqFromTreeSummarizedExperiment
+#' @export
+#'
+#' @author Leo Lahti and Tuomas Borman. Contact: \url{microbiome.github.io}
+#'
+#' @examples
+#' # Get TSE object
+#' data("GlobalPatterns")
+#' TSE <- GlobalPatterns
+#'
+#' # Create a phyloseq object from it
+#' phy <- makePhyloseqFromTreeSummarizedExperiment(TSE)
```

Comment:
For generating coy & paste examples and also generating nice pkgdown output, add a line printing the `phy`object.
Like this:

```
phy <- makePhyloseqFromTreeSummarizedExperiment(TSE)
phy
```

Reply:
Added